newperiod_components.cpp

```diff
@@ -140,12 +140,12 @@ void daccustodian::setCustodianAuths() {
             .accounts = accounts
     };
 
-    action(permission_level{accountToChange, "owner"_n},
+    action(permission_level{accountToChange, "active"_n},
```

Comment:
This should be owner still

Reply:
Do you mean this for each of the inline actions or just this one?

Reply:
Changed it back to `owner`